samples/tab-ui-templates/ts/teamsapp.yml

```diff
@@ -0,0 +1,85 @@
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
```

Comment:
Please remove all unused actions. Refer to https://github.com/formulahendry/Microsoft-Teams-Samples/blob/junhan/v3/samples/tab-ui-templates/ts/teamsapp.yml

Reply:
removed